### PR TITLE
Update version for py3 upgrade.

### PR DESCRIPTION
**Pull request by madhuBayy**
@ldko @somexpert 
Updated version for python 3 upgrade.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 Change Log
 ==========
+2.0.0
+-----
+
+* Upgraded to Python 3.
+
```
Comment (ldko) [inline CHANGELOG.md]: Could you also add lines to mention the changes reflected in these commits:
https://github.com/unt-libraries/aubreylib/commit/23a3979cc0c3a767b1f4ec6d4dee0c56aa4be020
https://github.com/unt-libraries/aubreylib/commit/264a6c17da4425d3a80998acca83792305592413

Comment (madhuBayy) [reply CHANGELOG.md]: will do.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 Change Log
 ==========
+2.0.0
+-----
+
+* Upgraded to Python 3.
+* Increased timeout to 6 seconds.
```
Comment (ldko) [inline CHANGELOG.md]: Out of the context of the commit this description is a little vague, maybe: "Increased `urlopen` timeout to 6 seconds when determining file system location."

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,11 @@
...
+* Escaped URL parts when forming regex.
```
Comment (ldko) [inline CHANGELOG.md]: Maybe "Escaped URL parts when forming regex to find characters that cause incorrect URL parsing."

**Comment by ldko** [comment]
2 more things I just noticed. We need to update the pyuntl dependency version in setup.py, and you should add your name to the Contributors section of the README :).

**Review by ldko — APPROVED** [review]
Looks good to me :+1: .

**Comment by somexpert** [comment]
LGTM :thumbsup:

**Pull request merged**